src/index.html

```diff
@@ -21,7 +24,352 @@
 
-  <body>
-    <h1>Catalog</h1>
+  <body class="page__body body">
+    <header class="body__header header">
+      <a
+        href="index.html"
+        class="header__logo"
+      >
+        <img
+          src="images/logo.png"
+          alt="Moyo"
+        />
+      </a>
+      <nav class="header__nav nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__links is-active"
+            >
+              Apple
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__links"
+            >
+              Samsung
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__links"
+            >
+              Smartphones
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__links"
+              data-qa="nav-hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__links"
+            >
+              Gadgets
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__links"
+            >
+              Tablets
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__links"
+            >
+              Photo
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__links"
+            >
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="body__main main">
+      <div
+        class="card"
+        data-qa="card"
+      >
+        <img
+          class="card__image"
+          src="images/imac.jpeg"
+          alt="imac"
+        />
+
+        <h3 class="card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h3>
+
+        <p class="card__code">Product code: 195434</p>
+
+        <div class="card__rating">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <p class="card__ratingReveiws">Reviews: 5</p>
+        </div>
+
+        <div class="card__price">
+          <p class="card__priceName">Price:</p>
+          <p class="card__priceAmount">$2,199</p>
+        </div>
+
+        <button
+          class="button"
+          data-qa="card-hover"
+        >
+          Buy
+        </button>
+      </div>
+
+      <div class="card">
```

Comment:
Use the semantic tag for card
```suggestion
      <article class="card">
```